src/main/java/org/wso2/carbon/module/swiftiso20022/utils/JsonToMt940Utils.java

```diff
@@ -0,0 +1,332 @@
+     */
+    private static String constructClosingBalance(RequestPayloadModel requestPayload) {
+        return requestPayload.getClosingBalanceDetails().getIndicator() +
+                requestPayload.getClosingBalanceDetails().getDate() +
+                requestPayload.getClosingBalanceDetails().getCurrency() +
+                formatAmountValue(requestPayload.getClosingBalanceDetails().getBalanceAmount());
+    }
+
+    /**
+     * Method to construct the available closing balance.
+     *
+     * @param requestPayload  Request Payload
+     * @return  Constructed available closing balance
+     */
+    private static String constructClosingAvailableBalance(RequestPayloadModel requestPayload) {
+        return requestPayload.getClosingAvailableBalanceDetails().getIndicator() +
+                requestPayload.getClosingAvailableBalanceDetails().getDate() +
+                requestPayload.getClosingAvailableBalanceDetails().getCurrency() +
+                formatAmountValue(requestPayload.getClosingAvailableBalanceDetails().getBalanceAmount());
+    }
+
+    /**
+     * Method to construct the forward closing balance.
+     *
+     * @param requestPayload  Request Payload
+     * @return  Constructed available closing balance
+     */
+    private static String constructForwardAvailableBalance(RequestPayloadModel requestPayload) {
+        return requestPayload.getForwardAvailableBalanceDetails().getIndicator() +
+                requestPayload.getForwardAvailableBalanceDetails().getDate() +
+                requestPayload.getForwardAvailableBalanceDetails().getCurrency() +
+                formatAmountValue(requestPayload.getForwardAvailableBalanceDetails().getBalanceAmount());
+    }
+
+    /**
+     * Method to construct the statement Lines.
+     *
+     * @param requestPayload   Request payload
+     * @return Constructed statement lines
+     */
+    private static List<String> constructStatementLines(RequestPayloadModel requestPayload) {
+        List<TransactionModel> transactions = requestPayload.getTransactions();
+        List<String> statementLines = new ArrayList<>();
+
+        StatementLineModel statementLineModel;
+        for (TransactionModel transaction : transactions) {
+            statementLineModel = new StatementLineModel();
+            statementLineModel.setValueDate(transaction.getDateTime());
+            statementLineModel.setEntryDate(transaction.getDateTime().substring(4));
+            statementLineModel.setTransactionIndicator(transaction.getTransactionIndicator());
+            statementLineModel.setFundCode(transaction.getCurrency().substring(2));
+            statementLineModel.setAmount(formatAmountValue(transaction.getAmount()));
+            statementLineModel.setTransactionType(transaction.getTransactionType());
+            statementLineModel.setCustomerReference(transaction.getCustomerReference());
+            statementLineModel.setBankReference("//" + transaction.getTransactionReference());
```

Comment:
Applicable for other places as well

```suggestion
            statementLineModel.setBankReference(ConnectorConstants.DOUBLE_SLASH + transaction.getTransactionReference());
```

Reply:
FIxed in https://github.com/wso2-extensions/mediation-swift-iso20022-module/pull/2/commits/befc8f745ba4f7a2cb0b949ac349d98c62e8ec5b